spread.yaml

```diff
@@ -222,7 +239,7 @@ suites:
         restore: |
             $TESTSLIB/reset.sh
             if [[ "$SPREAD_SYSTEM" != ubuntu-core-16-* ]]; then
-                apt-get purge -y snapd snap-confine ubuntu-core-launcher
+                apt-get purge -y snapd snap-confine ubuntu-core-launcher || true
```

Comment:
Why do we need the  `|| true` ? When does this fail? I would prefer to avoid it if we can help it so that we know if something can not get purged.